import_ldraw.py

```diff
@@ -820,7 +820,25 @@ def create_model(self, scale, context):
                     bpy.ops.mesh.select_all(action='SELECT')
 
                     # Add small gaps between each brick
-                    bpy.ops.transform.resize(value=(0.99, 0.99, 0.99))
+                    gap_width = 0.3
+                    obj_scale = cur_obj.scale * scale
+                    dim = cur_obj.dimensions
+
+                    # Checks whether the object isn't flat in a certain direction
+                    # to avoid division by zero.
+                    # Otherwise, the scale factor is proportional to the inverse of
+                    # the dimension so that the mesh shrinks a fixed distance
+                    # (determined by the gap_width and the scale of the object)
+                    # in every direction, creating a uniform gap.
+                    if dim.x != 0: fac_x = 1-gap_width*abs(obj_scale.x)/dim.x
```

Comment:
Although it may look a bit nicer this way, please put the statements on a new line as the Python style guide says.